### PR TITLE
limitlessled: Clarify v5 bridge usage.

### PR DESCRIPTION
**Pull request by janLo**
**Description:**

Since the support of v6 landed in HA the v5 support stopped working if
the config is not manually changed to the old defaults.

This is related to https://github.com/home-assistant/home-assistant/issues/6286

### CODE REVIEW

**Comment by aneisch** [comment]
👍  👍

**Comment by balloob** [comment]
If you create a PR against the current branch we will be able to update the site once merged. Else it will have to wait till the next release.

**Comment by janLo** [comment]
Ok @balloob. I cherrypicked my commit on top of `current` and forced-pushed it to the PR-branch.

File: source/_docs/installation.markdown [hunk 1]
```diff
@@ -10,9 +10,88 @@ footer: true
 redirect_from: /getting-started/installation/
 ---
 
-Home Assistant provides multiple ways to be installed. If you are a Raspberry Pi owner then the [Hassbian](/docs/hassbian/) is an easy and simple way to run home Assistant.
+Home Assistant provides multiple ways to be installed. A requirement is that you have [Python](https://www.python.org/downloads/) installed. For Windows we require at least **Python 3.5** and for other operating systems at least **Python 3.4.2**.
```
Comment (Landrash) [inline source/_docs/installation.markdown]: These changes has nothing to do in this PR

File: source/_docs/installation/archlinux.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (Landrash) [inline source/_docs/installation/archlinux.markdown]: These changes has nothing to do in this PR.  Submit as a separate PR.

File: source/_docs/installation/armbian.markdown [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (Landrash) [inline source/_docs/installation/armbian.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_docs/installation/centos.markdown [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (Landrash) [inline source/_docs/installation/centos.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_docs/installation/fedora.markdown [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (Landrash) [inline source/_docs/installation/fedora.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_docs/installation/macos.markdown [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (Landrash) [inline source/_docs/installation/macos.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_docs/installation/troubleshooting.markdown [hunk 7]
```diff
@@ -38,9 +38,6 @@ On a Debian system, install the Python 3 YAML library by `sudo apt-get install p
 #### {% linkable_title distutils.errors.DistutilsOptionError: must supply either home or prefix/exec-prefix -- not both %}
 This is a known issue if you're on a Mac using Homebrew to install Python. Please follow [these instructions](https://github.com/Homebrew/brew/blob/master/docs/Homebrew-and-Python.md#note-on-pip-install---user) to resolve it.
 
-#### {% linkable_title CentOS and Python 3 %}
```
Comment (Landrash) [inline source/_docs/installation/troubleshooting.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_docs/installation/windows.markdown [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Landrash) [inline source/_docs/installation/windows.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/_includes/asides/docs_navigation.html [hunk 9]
```diff
@@ -5,13 +5,8 @@ <h1 class="title delta">Topics</h1>
       <li>
```
Comment (Landrash) [inline source/_includes/asides/docs_navigation.html]: These changes has nothing to do in this PR. Submit as a separate PR.

File: source/getting-started/getting-started-alternatives.markdown [hunk 10]
```diff
@@ -1,29 +0,0 @@
----
```
Comment (Landrash) [inline source/getting-started/getting-started-alternatives.markdown]: These changes has nothing to do in this PR. Submit as a separate PR.

**Comment by janLo** [comment]
@Landrash  I don't know why github decided to put all these commit in the PR - probably an error in their change detection. I only wanted e7b280f to be merged and the branch the PR came from was this commit on top of an up-to-date `current` (as @balloob suggested in his comment https://github.com/home-assistant/home-assistant.github.io/pull/2171#issuecomment-283869084). So please only merge e7b280f and ignore the rest.

**Comment by Landrash** [comment]
@fabaff Can you help me to only merge the one requested above?

**Comment by janLo** [comment]
```
git fetch git fetch origin pull/2171/head:limitless-v5-clarify
git cherry-pick e7b280f0d6468664f97f0d56b97e652e69a86a0b
```
Should do the trick.

**Comment by balloob** [comment]
```
error: could not apply e7b280f0d... limitlessled: Clarify v5 bridge usage.
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
```

**Comment by Landrash** [comment]
Replaced pr with #2569 .
